### PR TITLE
Example for index multi-document support

### PR DESCRIPTION
**Pull request by luizgpsantos**
The previous docs provided instructions on how to create the _doc payload field. However, it's not trivial and an example can help to create it.

### CODE REVIEW

**Comment by colings86** [comment]
@russcam are you still wanting to progress this PR or should it be closed?

**Comment by luizgpsantos** [comment]
@elasticmachine run elasticsearch-ci/packaging-sample-unix-archives

**Comment by luizgpsantos** [comment]
@elasticmachine update branch

**Comment by luizgpsantos** [comment]
Hi @colings86, thanks for asking. I'd like to proceed with this pull request. The tests failed for some reason but succeed when I updated the branch. Could you provide feedback or merge if all good?

File: x-pack/docs/en/watcher/actions/index.asciidoc [hunk 1]
```diff
@@ -73,5 +73,34 @@ When a `_doc` field exists, if the field holds an object, it is extracted and in
...
+    "transform": {
+      "script": """
+      def documents = ctx.payload.hits.hits.stream()
+        .map(hit -> [
+          "_index": "my-index", <1>
+          "_id": hit._id, <2>
+          "severity": "Sev: " + hit._source.severity <3>
+        ])
+        .collect(Collectors.toList());
+      return [ '_doc' : documents]; <4>
```
Comment (jrodewig) [inline x-pack/docs/en/watcher/actions/index.asciidoc:94]:
```suggestion
      return [ "_doc" : documents]; <4>
```

**Review by jrodewig — APPROVED** [review]
Hi @luizgpsantos.

This looks good to me. I did make a minor suggestion. If that's okay with you, I'll merge and backport this.

Thanks for the contribution!

**Comment by luizgpsantos** [comment]
Hi @jrodewig, sure! Thanks for the suggestion, I just committed it.

**Comment by jrodewig** [comment]
### Backport commits
master eb9de19
7.x 37e2bb705721eb29415dba4c7635b20aaca3d29b 
7.8 eece3ee 
7.7 120ddb0

**Pull request merged**